network/policy/policy_windows.go

```diff
@@ -108,3 +126,235 @@ func SerializeOutBoundNATPolicy(policies []Policy, epInfoData map[string]interfa
 
 	return nil, fmt.Errorf("OutBoundNAT policy not set")
 }
+
+// GetPolicyType parses the policy and returns the policy type
+func GetPolicyType(policy Policy) CNIPolicyType {
+	// Check if the type is OutBoundNAT
+	var dataOutBoundNAT KVPairOutBoundNAT
+	if err := json.Unmarshal(policy.Data, &dataOutBoundNAT); err == nil {
+		if dataOutBoundNAT.Type == OutBoundNatPolicy {
+			return OutBoundNatPolicy
+		}
+	}
+
+	// Check if the type is Route
+	var dataRoute KVPairRoute
+	if err := json.Unmarshal(policy.Data, &dataRoute); err == nil {
+		if dataRoute.Type == RoutePolicy {
+			return RoutePolicy
+		}
+	}
+
+	// Check if the type if Port mapping / NAT
+	var dataPortMapping KVPairPortMapping
+	if err := json.Unmarshal(policy.Data, &dataPortMapping); err == nil {
+		if dataPortMapping.Type == PortMappingPolicy {
+			return PortMappingPolicy
+		}
+	}
+
+	// Return empty string if the policy type is invalid
+	log.Printf("Returning policyType INVALID")
+	return ""
+}
+
+// SerializeHcnSubnetVlanPolicy serializes subnet policy for VLAN to json.
+func SerializeHcnSubnetVlanPolicy(vlanID uint32) ([]byte, error) {
+	vlanPolicySetting := &hcn.VlanPolicySetting{
+		IsolationId: vlanID,
+	}
+
+	vlanPolicySettingJSON, err := json.Marshal(vlanPolicySetting)
+	if err != nil {
+		return nil, err
+	}
+
+	vlanSubnetPolicy := &hcn.SubnetPolicy{
+		Type:     hcn.VLAN,
+		Settings: vlanPolicySettingJSON,
+	}
+
+	vlanSubnetPolicyJSON, err := json.Marshal(vlanSubnetPolicy)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
if GetOutBoundNatExceptionList returns empty list, there's not need to check for nil in the following if statement